uds/utilities/helpers.py

```
@@ -0,0 +1,41 @@
+def validate_time(value: TimeMillisecondsAlias, accept_zero: bool = True) -> None:
+    """
+    Validate time value.
+
+    :param value: Time value to check.
+    :param accept_zero: Whether zero is acceptable value.
+
+    :raise TypeError: Provided value is not int or float type.
+    :raise ValueError: Provided value is a negative number or equal zero (accept_zero=False).
+    """
+    if not isinstance(value, (int, float)):
+        raise TypeError(f"Time value must be int or float type. Actual type: {type(value)}.")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)